src/app/(common)/reviews/_components/ReviewsAverage.tsx

```
@@ -0,0 +1,31 @@
+      <div className="flex w-full max-w-[610px] justify-between">
+        <div className="flex flex-col items-center justify-center gap-2 pr-5">
+          <div className="flex items-center justify-center text-xl font-semibold md:text-2xl">
+            <span>{averageScore}</span>
+            <span className="text-gray-400">/5</span>
+          </div>
+          <Score score={averageScore} />
+        </div>
+
+        <div className="flex max-w-[302px] flex-1 flex-col gap-1">
+          {stars.map((count, index) => (
+            <ScoreProgressBar score={5 - index} stars={count} total={total} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)